### DISCUSSION TITLE
Proposal submission details for `GSoC`

### DISCUSSION THREAD

**Discussion by Abinashbunty** [wagtail/gsoc#61] [General]
Hi @wagtail members. 😄 I was about to submit my proposal for GSoC. But certain parts of it are a bit confusing for me. Here are those:

1. For the `Proposal Summary` section, what should I exactly fill in? Is this something I can be guided with or is this something I have to do on my own? I mean, I have briefly mentioned about the topics that I have researched about in  #49 Should I just elaborate the Research section of the `README` file in my PR and submit for proposal?
2. The `Project size` field expects values `Medium` or `Large`. But in the project ideas files, the size is mentioned in hours. So what should be the right value for my application? 😅 
3. A PDF also has to be uploaded. Should it contain basically the same thing that I fill in the text fields? Or should it contain some extra information?

Apologies for so many questions. 😅 It's just that I am unsure of what exact details I have to fill and I want to increase my probability of acceptance as well.

**Comment by thibaudcolas** [comment]
👋 good questions here! No need to apologise, I wish more people used the opportunity to ask questions in this forum.

1. Proposal summary: This is meant to be a one/two-sentence description of your proposal. It’s just for the reviewers to have an easier time distinguishing between different proposals. Keep it simple, and **short**.
2. Project size: see [How much time does GSoC participation take?](https://developers.google.com/open-source/gsoc/faq#how_much_time_does_gsoc_participation_take)
3. The PDF is where your proposal is meant to be. To make it easier, I have created a template: https://wagtail.org/gsoc-template. There is no requirement to follow this to the letter, but we do need you to send us the document _as a Google Docs_, _with the right sharing settings_ if you want feedback ahead of the application deadline.

Keep the questions coming!

**Reply by Nathaniel330** [reply]
Hello!


I just found out about this "Discussions" tab. I thought only the Pull requests tab has activity because it has numbers beside it 🤣

Regarding the proposal, what if I do not have one yet? Can I just put the details for **Greener coding: Wagtail’s climate impact**?

Of course I think creating a proposal would be better, but I am still currently learning Django and I still haven't touched Wagtail yet.

Is my own proposal really necessary or can I just join the current project idea?


Thanks for the response!

**Reply by thibaudcolas** [reply]
@Nathaniel330 see https://google.github.io/gsocguides/student/writing-a-proposal which is pretty clear. Note a basic understanding of how GSoC works is a fundamental requirement for working with us, so I’d recommend you start there.

This "Discussions" tab is the only mean of contact listed on our GSoC org listing, and is pretty clearly indicated in this project’s README.

**Reply by Nathaniel330** [reply]
Thank you so much! I've been doing too much all at once my thoughts are getting messed up. I apologize for all the troubles I caused.
